drivers/bridge/bridge.go

```diff
@@ -145,6 +145,26 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// Conflict check if two NetworkConfiguration objects overlap in the multinetwork
+func (c *NetworkConfiguration) Conflict(o *NetworkConfiguration) bool {
+	if o == nil {
+		return false
+	}
+
+	// Also empty, becasue only one network with empty name is allowed
```

Comment:
There shouldnt be any network with a bridgename that is empty. That should be marked as an error.
Even for the default case, we must force the client to pass the value (say `docker0` in this case)